### PR TITLE
Initial version for sample.

### PR DESCRIPTION
**Pull request by huxuan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,26 @@ result = CF.face.detect(img_url)
...
+
+A Python SDK sample built with wxPython is also provided, before execution,
+please install all components listed below.
+
+### Sample Prerequisite
+
+- [Python 2.7](https://www.python.org/downloads/) (only Python 2 supported due
+  to limitation of wxPython)
+- [wxPython](https://wxpython.org/)
+- [cognitive_face package](https://pypi.python.org/pypi/cognitive_face)
```
Comment (wangjun-microsoft) [inline README.md]: better to name it as "Microsoft_projectoxford_face", to be consistent with other packages.

Comment (wangjun-microsoft) [inline README.md]: better to name it as "microsoft_projectoxford_face", to be consistent with other packages.

Comment (wangjun-microsoft) [inline README.md]: "microsoft_projectoxford_face" instead?

**Comment by huxuan** [comment]
Done for remove `TODO` section in `README`.

**Comment by huxuan** [comment]
Several logic and view bugs reported by Yuan Xing have been fixed.

**Comment by huxuan** [comment]
Several logic and view bugs reported by Yuan Xing have been fixed.

**Pull request merged**